services/gemma-local/src/adapters/audio_processor_adapter.py

```
@@ -0,0 +1,33 @@
+        audio_base64: str,
+        context_prompt: Optional[str] = None,
+        use_chunking: bool = False,
+        request_id: Optional[str] = None
+    ) -> tuple:
+        """Process base64 audio data"""
+        return await self.legacy_processor.process_audio_base64(
+            audio_base64=audio_base64,
+            context_prompt=context_prompt,
+            use_chunking=use_chunking,
+            request_id=request_id
+        )
```

Ruff (0.13.1)

No newline at end of file

Add trailing newline

(W292)